docs/user.md

```diff
@@ -103,6 +90,15 @@ Here is a `MachineSet` YAML snippet on how to configure the label and taint as w
           effect: NoSchedule
 ```
 
+### Development Environment <a name="development-env"></a>
```

Comment:
@bostrt I am not sure if we need to keep this section. A file `dev.md` is designed for devs, and this option should never run in production (VCSP results).

I also think that we should remove the `--dedicated` flag, and introduce it to env var like `DEV_DEDICATED=false`, so we can propagate this somewhere (existing CM). So we can prevent human-errors.

Would love to hear from you.

Reply:
Sounds good to me :+1:

Reply:
https://github.com/redhat-openshift-ecosystem/provider-certification-tool/pull/29/commits/e484eedb2cfe70511d99ee4020724c1540b0c126